src/js/theme/input-active.js

```diff
@@ -0,0 +1,33 @@
+// Set it and forget it.
+const body = document.querySelector( 'body' );
+
+// Add the active class to text inputs on focus.
+body.addEventListener( 'focus', event => {
+	if ( event.target.classList.contains( 'input--text' ) ) {
+		event.target.closest( '.gfield' ).classList.add( 'input--active' );
+	}
+}, true );
+
+// Remove the active class from text inputs on blur if no value entered.
+body.addEventListener( 'blur', event => {
+	if ( event.target.classList.contains( 'input--text' ) && '' === event.target.value ) {
+		event.target.closest( '.gfield' ).classList.remove( 'input--active' );
+	}
+}, true );
+
+// Add the active class to selects once option has been chosen.
+body.addEventListener( 'change', event => {
+	if ( event.target.classList.contains( 'input--select' ) && '' !== event.target.value ) {
+		event.target.closest( '.gfield' ).classList.add( 'input--active' );
+	}
+}, true );
+
+// When the form is rendered, check if any fields have a value, if so give them the active class.
+// Using jQuery because it's the only way to access the 'gform_post_render' event.
+jQuery( document ).on( 'gform_post_render', function() {
+	jQuery( '.input--text' ).each( function() {
+		if ( jQuery( this ).val() !== '' ) {
+			jQuery( this ).closest( '.gfield' ).addClass( 'input--active' );
+		}
+	} );
+} );
```

Comment:
`gform_post_render` gives you two parameters, the gform ID and the current page of the form (the latter is not relevant for us this time). This code block will loop through _all_ `.input--text` fields in the DOM.
We should target those `.input--text` fields inside of the form by targeting the ID of the form by using the parameter `form_id` parameter.